CalibTracker/SiStripCommon/interface/Book.h

```diff
@@ -104,7 +104,7 @@ class Book {
 	     uint_t NbinsY, double_t Ylow, double_t Yup,
 	     uint_t NbinsZ, double_t Zlow, double_t Zup, double_t W=1 )
   { 
-    BOOST_FOREACH(string_t name, std::make_pair(names.begin(),names.end())) {
+    for(auto const& name : names) {
```

Comment:
@guitargeek are you sure that those loops are equivalent? `std::make_pair` will return a pair of first and last elements while the new version will iterate over every value in `names`.

Reply:
Loops are indeed equivalent as discussed above.